### ISSUE TITLE
Bags position resets after using Jeeves or Bank

### ISSUE THREAD

**Issue by Jaidy88** [Cidan/BetterBags#441]
### Addon Version

v0.1.67

### World of Warcraft Version

Retail

### Describe the bug

I noticed that the bag doesnt keep the position i after using Jeeves, or Repair Bot. i had hoped that it would finally be fixed with version v0.1.67, but unfortunately not

### Steps to reproduce

open bag, move to specific position, use Jeeves or Repairbot, go away from them and open bags again now you see that the bag doesnt keep the postion:

https://github.com/Cidan/BetterBags/assets/47262722/19c18b80-d4e4-4e16-80cb-f38263568a9f

**Comment by Jaidy88** [comment]
jeeves seems to cause the problem because of the "merge stacks" function

**Comment by Cidan** [comment]
Found the bug and this is fixed in the next release, due out later today.

**Comment by Jaidy88** [comment]
> Found the bug and this is fixed in the next release, due out later today.

:( it isn't fixed..... do you need the same merge settings for the bank and the pocket so that the window does not move every time?

[https://gyazo.com/76c56cc637d051cc1879079f077cd52e](url)

**Comment by Cidan** [comment]
Hm, I can't reproduce it on my end. Let me try again a bit later tonight and see if I can hunt it down.

**Comment by AMaribo** [comment]
It seems like the issue is that when the vendor window opens up, the stacks unstack and extend the bag another row (or more).
The bag is positioned such that, with stacks merged, it fits exactly at the top of the screen.
Adding another row forces the bag to extend downward, which updates the anchor to be at a lower position, so that when the vendor window is closed, the bags are now lower on the screen at the new anchor point.

**Comment by Cidan** [comment]
Ah, this is where a configurable anchor will fix this -- this is on the list to implement.
